src/react-components/src/index.ts

```diff
@@ -1,30 +1,30 @@
+export { default as Image } from "./components/Image/Image";
 export {
     ButtonIconPositions,
     ButtonTypes,
-} from "./components/01-atoms/Button/ButtonTypes";
+} from "./components/Button/ButtonTypes";
 export { default as SectionName } from "./components/SectionName/SectionName";
-export { default as BodyText } from "./components/01-atoms/Text/Text/BodyText";
-export { default as UnderlineLink } from "./components/01-atoms/Links/UnderlineLink";
-export { default as BasicLink } from "./components/01-atoms/Links/BasicLink";
-export { default as IconLink } from "./components/01-atoms/Links/IconLink";
-export { default as Heading } from "./components/01-atoms/Text/Headings/Heading";
-export { default as SectionTitle } from "./components/Headings/SectionTitle";
-export { default as Breadcrumbs } from "./components/02-molecules/Menus/Breadcrumbs/Breadcrumbs";
-export { default as Pagination } from "./components/02-molecules/Menus/Pagination/Pagination";
-export { default as Dropdown } from "./components/02-molecules/Dropdown/Dropdown";
-export { default as EditionCard } from "./components/02-molecules/Cards/EditionCard";
-export { default as SearchResultItem } from "./components/02-molecules/Cards/SearchResultItem";
-export { default as UnorderedList } from "./components/03-organisms/List/UnorderedList";
-export { HeroTypes } from "./components/03-organisms/Hero/HeroTypes";
-export { default as Hero } from "./components/03-organisms/Hero/Hero";
-export { default as SearchPromo } from "./components/03-organisms/SearchPromo/SearchPromo";
-export { default as HeaderWithSearch } from "./components/03-organisms/Headers/RNHeaderWithSearch";
-export { default as DateRangeForm } from "./components/01-atoms/Forms/DateRangeForm";
-export { default as Checkbox } from "./components/01-atoms/Forms/Checkbox";
-export { default as Label } from "./components/01-atoms/Forms/Label";
-export { default as Input } from "./components/01-atoms/Forms/Input";
-export { default as Accordion } from "./components/03-organisms/Accordion/Accordion";
+export { default as BodyText } from "./components/BodyText/BodyText";
+export { default as UnderlineLink } from "./components/UnderlineLink/UnderlineLink";
+export { default as BasicLink } from "./components/BasicLink/BasicLink";
+export { default as IconLink } from "./components/IconLink/IconLink";
+export { default as Heading } from "./components/Heading/Heading";
+export { default as SectionTitle } from "./components/SectionTitle/SectionTitle";
+export { default as Breadcrumbs } from "./components/Breadcrumbs/Breadcrumbs";
+export { default as Pagination } from "./components/Pagination/Pagination";
+export { default as Dropdown } from "./components/Dropdown/Dropdown";
+export { default as EditionCard } from "./components/EditionCard/EditionCard";
+export { default as SearchResultItem } from "./components/SearchResultItem/SearchResultItem";
+export { default as UnorderedList } from "./components/List/UnorderedList";
```

Comment:
Should this be renamed from `"./components/List/UnorderedList";` to `"./components/UnorderedList/UnorderedList";` to align with the other naming conventions?

Reply:
Renamed, good catch